scripts/mkcloudhost/generate-radvd-conf

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+cloudfunc=~/cloud.d/cloudfunc
+test -e $cloudfunc || cloudfunc=$(dirname $(readlink -e $BASH_SOURCE))/cloudfunc
+. $cloudfunc
+
+host_letter=$(hostname | cut -b5)
+
+snippet=$(mktemp)
+cat <<EOF >$snippet
+interface v${host_letter}<NUM>br
+{
+  AdvSendAdvert on;
+  MinRtrAdvInterval 30;
+  MaxRtrAdvInterval 100;
+  AdvManagedFlag off;
+  AdvOtherConfigFlag on;
```

Comment:
This option is telling the nodes that use DHCP6 for everything but an address. Which sounds good if we ever need/want to advertise a DNS server etc. But this could just was easily be 'off' especially if we wont be providing this on this network. The the admin node itself would deploy a dhcpd server after the "instcrowbar" step has completed. So not sure how that will play out with further steps.

Not necessarily saying we should change this.. but there might be a chance we may need to tweak things one get start deploying nodes. I.e DHCPv6 for the admin network when nodes are deployed in the cloud way need a different RA.
But as we've mentioned this patch is required to get the admin node up with an IP address we'll know.

Reply:
Oh! This probably totally explains an issue I'm seeing with current admin node deployments. I'm going to set this to off and see what happens. Thanks for the hint, I seriously doubt we want a dhcpd on the mkch* hosts.